synapse/tests/test_model_orgs.py

```diff
@@ -1,52 +1,180 @@
+import synapse.exc as s_exc
 import synapse.common as s_common
-
-import unittest
-raise unittest.SkipTest()
-
-import synapse.lib.types as s_types
-
-from synapse.tests.common import *
-
-class OrgTest(SynTest, ModelSeenMixin):
-
-    def test_model_ou_has(self):
-        with self.getRamCore() as core:
-            org_guid = 32 * '0'
-            org_tufo = core.formTufoByProp('ou:org', org_guid, name='The Vertex Project')
-            orgval = org_tufo[1].get('ou:org')
-
-            node = core.formTufoByProp('ou:org:has', (orgval, ('inet:fqdn', 'vertex.link')))
-            self.ge(node[1].get('node:created'), 1519852535218)
-            self.eq(node[1].get('ou:org:has'), '03870dc800bc21c7c594a900ae65f5cb')
-            self.eq(node[1].get('ou:org:has:org'), orgval)
-            self.eq(node[1].get('ou:org:has:xref'), 'inet:fqdn=vertex.link')
-            self.eq(node[1].get('ou:org:has:xref:prop'), 'inet:fqdn')
-            self.eq(node[1].get('ou:org:has:xref:node'), '42366d896b947b97e7f3b1afeb9433a3')
-
-            self.none(core.getTufoByProp('node:ndef', '42366d896b947b97e7f3b1afeb9433a3'))  # Not automatically formed
-            core.formTufoByProp('inet:fqdn', 'vertex.link')
-            fqdnfo = core.getTufoByProp('node:ndef', '42366d896b947b97e7f3b1afeb9433a3')
-            self.eq(fqdnfo[1].get('inet:fqdn'), 'vertex.link')
-
-    def test_model_org_has_alias(self):
-        with self.getRamCore() as core:
-            iden = 32 * '0'
-
-            node = core.formTufoByProp('ou:hasalias', (iden, 'cools'))
-            self.eq(node[1].get('ou:hasalias'), '77ff3cd31931a67b658af31260ade638')
-            self.eq(node[1].get('ou:hasalias:org'), iden)
-            self.eq(node[1].get('ou:hasalias:alias'), 'cools')
-
-            node = core.formTufoByProp('ou:hasalias', (iden, 'b4dZ'))
-            self.eq(node[1].get('ou:hasalias'), '4be15b22e4081e102d6c8201ca26f28f')
-            self.eq(node[1].get('ou:hasalias:org'), iden)
-            self.eq(node[1].get('ou:hasalias:alias'), 'b4dz')
-
-            self.check_seen(core, node)
-
-            self.len(2, core.getTufosByProp('ou:hasalias:org', iden))
-
-            self.raises(BadTypeValu, core.formTufoByProp, 'ou:hasalias', (iden, 'wee!!!'))
+import synapse.tests.common as s_test
+
+# TODO
```

Comment:
Did you want to add these tests before merging?

Reply:
added one of them; the other requires more datamodel-node formation changes.